docs/source/developer/documentation.rst

```diff
@@ -69,6 +70,7 @@ The following sections of the user interface should be in title case, directly q
 - Files tab
 - Running panel
 - Tabs panel
+- Single-Document Mode
```

Comment:
I think the dash here seems grammatically correct. :+1:

Reply:
The dash is grammatically correct and technically what we have in the UI